Modules/CalcOffence-3_0.lua

```diff
@@ -1648,7 +1648,10 @@ function calcs.offence(env, actor, activeSkill)
 				local mult = skillModList:Sum("BASE", dotCfg, "PhysicalDotMultiplier", "BleedMultiplier")
 				local effectMod = calcLib.mod(skillModList, dotCfg, "AilmentEffect")
 				local rateMod = calcLib.mod(skillModList, cfg, "BleedFaster")
-				output.BleedDPS = baseVal * effectMod * rateMod * effMult
+				local maxStacks = skillModList:Override(cfg, "BleedStacksMax") or skillModList:Sum("BASE", cfg, "BleedStacksMax")
+				local configStacks = enemyDB:Sum("BASE", nil, "Multiplier:BleedStacks")
```

Comment:
I suggest renaming both variables `maxStacks` and `configStacks` to something that incorporates "bleed", as such generic names could lead to further changes needed once a new, stack-based game mechanic is introduced. Maybe change them to `bleedStacksMax` and `bleedStacksConfig`?

Reply:
I made it exactly like how impale is handled. Also they are in an if that is only for bleed calculation.